sql/2021/security/cryptominer_usage.sql

```diff
@@ -0,0 +1,16 @@
+#standardSQL
+SELECT
+  LEFT(_TABLE_SUFFIX, 10) AS month,
+  IF(ENDS_WITH(_TABLE_SUFFIX, '_desktop'), 'desktop', 'mobile') AS client,
+  COUNT(0) AS freq
```

Comment:
Should we look at this as a percentage? As our number of URLs changes each month the absolute value can be deceiving.